RadarSDK.xcodeproj/project.pbxproj

```diff
@@ -949,7 +949,6 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = 3.9.6;
```

Comment:
this is the package settings, which overwrites the project settings, we should just have the version set for project settings which will be resolved everywhere to the single correct marketing version